src/screens/CreatePost/index.js

```diff
@@ -833,7 +813,6 @@
             topics={listTopic}
             chatTopics={listTopicChat}
             onClose={() => sheetTopicRef.current.close()}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The commented-out `saveOnClose` prop has been removed from the `SheetTopic` component. If this functionality is no longer needed, this is a good change for code cleanliness.
```diff
- // saveOnClose={(v, chatTopic) => onSaveTopic(v, chatTopic)}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->